corehq/apps/builds/management/commands/add_commcare_build.py

```diff
@@ -1,26 +1,91 @@
-        parser.add_argument('version')
-        parser.add_argument('build_number')
+        parser.add_argument('build_path', nargs='?')
+        parser.add_argument('version', nargs='?')
+        parser.add_argument('build_number', nargs='?')
+        parser.add_argument(
+            '-l',
+            '--latest',
+            action='store_true',
+            help="add the latest remote CommCare build version"
+        )
 
     def handle(self, build_path, version, build_number, **options):
-        try:
-            build_number = int(build_number)
-        except ValueError:
-            raise CommandError("Build Number %r is not an integer" % build_number)
-
-        try:
-            CommCareBuild.create_from_zip(build_path, version, build_number)
-        except Exception as e:
-            raise CommandError("%s" % e)
-        self.stdout.write('Build %s #%s created\n' % (version, build_number))
-        self.stdout.write('You can see a list of builds at [your-server]/builds/\n')
+        if options.get('latest'):
+            _create_build_from_latest_remote_version()
+        else:
+            if build_path and version and build_number:
+                try:
+                    build_number = int(build_number)
+                except ValueError:
+                    raise CommandError("Build Number %r is not an integer" % build_number)
+
+                try:
+                    CommCareBuild.create_from_zip(build_path, version, build_number)
+                except Exception as e:
+                    raise CommandError("%s" % e)
+                self.stdout.write('Build %s #%s created\n' % (version, build_number))
+                self.stdout.write('You can see a list of builds at [your-server]/builds/\n')
+            else:
+                raise CommandError("<build_path>, <version> or <build_number> not specified!")
+
+
+def _create_build_from_latest_remote_version():
+    version = _get_latest_commcare_build_version()
+    # arbitrary build_number
+    build_number = random.randint(0, 100)
+
+    CommCareBuild.create_without_artifacts(version, build_number)
+    _update_commcare_build_menu(version, build_number)
+
+
+def _get_latest_commcare_build_version():
+    repo = Github().get_organization('dimagi').get_repo("commcare-android")
+    latest_release_tag = repo.get_latest_release().tag_name
+
+    return latest_release_tag.split('commcare_')[1]
+
+
+def _update_commcare_build_menu(version, build_number):
+    doc = CommCareBuildConfig.fetch()
+    _add_build_menu_item(doc, version, build_number)
+    _update_default_build_spec_to_version(doc, version)
+
+    CommCareBuildConfig.get_db().save_doc(doc)
+    CommCareBuildConfig.clear_local_cache()
+
+
+def _add_build_menu_item(doc, version, build_number):
+    build_menu_items = doc.menu
+
+    build_menu_item = next(
+        (build_item for build_item in build_menu_items if build_item.build.version == version),
+        None
+    )
+    if build_menu_item is None:
```

Comment:
if you only create a new build when the version changes, you can drop this check